Microsoft.Toolkit.Uwp.UI.Controls.Graph/PlannerTaskList/PlannerTaskList.cs

```diff
@@ -0,0 +1,346 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using Microsoft.Graph;
+using Microsoft.Toolkit.Services.MicrosoftGraph;
+using Windows.UI.Popups;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Graph
+{
+    /// <summary>
+    /// The PlannerTaskList Control displays a simple list of Planner tasks.
+    /// </summary>
+    [TemplatePart(Name = ControlTasks, Type = typeof(ListView))]
+    [TemplatePart(Name = ControlInput, Type = typeof(TextBox))]
+    [TemplatePart(Name = ControlAdd, Type = typeof(Button))]
+    [TemplateVisualState(GroupName = MobileVisualStateGroup, Name = MobileVisualState)]
+    public partial class PlannerTaskList : Control
+    {
+        private Dictionary<string, string> _userCache = new Dictionary<string, string>();
+        private List<PlannerTaskViewModel> _allTasks = new List<PlannerTaskViewModel>();
+        private TextBox _input;
+        private ListView _list;
+        private Button _add;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PlannerTaskList"/> class.
+        /// </summary>
+        public PlannerTaskList()
+        {
+            this.DefaultStyleKey = typeof(PlannerTaskList);
+        }
+
+        /// <inheritdoc/>
+        protected async override void OnApplyTemplate()
+        {
+            ClearTasks();
+
+            if (_list != null)
+            {
+                _list.ItemClick -= List_ItemClick;
+                _list.Tapped += List_Tapped;
+                if (!IsWindowsPhone)
+                {
+                    _list.PreviewKeyUp -= List_PreviewKeyUp;
+                }
+            }
+
+            if (_add != null)
+            {
+                _add.Click -= Add_Click;
+            }
+
+            base.OnApplyTemplate();
+            if (GetTemplateChild(ControlTasks) is ListView list)
+            {
+                _list = list;
+                _list.ItemClick += List_ItemClick;
+                _list.Tapped += List_Tapped;
+                if (!IsWindowsPhone)
+                {
+                    _list.PreviewKeyUp += List_PreviewKeyUp;
+                }
+            }
+
+            if (GetTemplateChild(ControlAdd) is Button add)
+            {
+                _add = add;
+                _add.Click += Add_Click;
```

Comment:
Maybe implement add via commanding

Reply:
@skendrot We heard two different voices on this, in one of our control in the toolkit, we removed the DelegatedCommand and use event handler instead as it's said that command is not recommended in the controls project, so we just simply followed that path, is there any change now?